### DISCUSSION TITLE
weeklies/13

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#164] [Comments]
# articles/weekly-13

本文为周刊第13期，前期的周刊参见：周刊系列。

https://hutusi.com/articles/weekly-13

**Comment by senlypan** [comment]
GeekPlux 的周刊，我看竹白的链接是失效了，可以使用 geekplux.com

**Reply by hutusi** [reply]
谢谢，已修改。不过这位也已停更。
